backend/src/main/scala/com/softwaremill/adopttapir/starter/api/StarterApi.scala

```diff
@@ -0,0 +1,48 @@
+package com.softwaremill.adopttapir.starter.api
+
+import cats.data.{EitherT, NonEmptyList}
+import cats.effect.IO
+import cats.implicits.toBifunctorOps
+import com.softwaremill.adopttapir.Fail
+import com.softwaremill.adopttapir.http.Http
+import com.softwaremill.adopttapir.starter._
+import com.softwaremill.adopttapir.util.ServerEndpoints
+import fs2.io.file.Files
+import io.circe.generic.auto._
+import sttp.capabilities.fs2.Fs2Streams
+import sttp.tapir.generic.auto._
+
+class StarterApi(http: Http, starterService: StarterService) {
+  import http._
+
+  private val starterPath = "starter.zip"
+
+  private val starterEndpoint = baseEndpoint.get
+    .in(starterPath)
+    .in(query[String]("tapirVersion"))
+    .in(jsonBody[StarterRequest])
```

Comment:
GET requests shouldn't have a body -> this should be a POST probably

Reply:
done